File: SistemaCompra/Back/src/SistemaCompra.Domain/ItemPedido.cs
namespace SistemaCompra.Domain
{
    public class ItemPedido
    {
        public int Id { get; set; }
        public int IdPedido { get; set; }
        public int IdProduto { get; set; }
        public int QtdeProduto { get; set; }
        public double PrecoUnit { get; set; }
         public int itemCotacaoId { get; set; }
         public ItemCotacao itemCotacao { get; set; }
        public int PedidoId { get; set; }
         public Pedido Pedido { get; set; }
    }
}

File: Front/ProEventos-App/src/app/models/ItemPedido.ts
import { ItemCotacao } from './ItemCotacao';
import { Pedido } from './Pedido';

export interface ItemPedido {
   id: number;
   idPedido: number;
   idProduto: number;
   qtdeProduto: number;
   precoUnit: number;
   totalItem: number
   itemCotacaoId: number;

}
